lib/msal-core/src/cache/AuthCache.ts

```diff
@@ -70,7 +70,7 @@ export class AuthCache extends BrowserStorage {// Singleton
             JSON.parse(key);
             return key;
         } catch (e) {
-            if (key.indexOf(`${Constants.cachePrefix}`) === 0 || key.indexOf(PersistentCacheKeys.ADAL_ID_TOKEN) === 0) {
+            if (key.startsWith(`${Constants.cachePrefix}`) || key.startsWith(Constants.adalIdToken)) {
```

Comment:
key.startsWith isn't supported on all browsers, which is why we used indexOf.

Reply:
Done.